### PR TITLE
Add lesson19

### PR DESCRIPTION
**Pull request by haraldF**

### CODE REVIEW

File: lesson19.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Array access is index based (first index is `0`)
+
+Object access is name based (key-value pairs)
+
+```js
+let arr = [ 'Pizza', 'Muffin' ];
+let obj = {
+    favoriteFood = 'Pizza',
+    favoriteSweet = 'Muffin'
```
Comment (carlotrimarchi) [inline lesson19.md]: 😱

File: lesson19.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+### DOM and Arrays
+
+Let's say we have the following HTML:
+
+```html
+<ul id="todoList">
+  <li>Eat
+  <li>Sleep
+  <li>Work
+</ul>
```
Comment (carlotrimarchi) [inline lesson19.md:63]: isn't it better to properly close them? I know that it's valid html, but I'm sure a few people would ask about it and kind of diverge

File: lesson19.md (hunk 1 above)
Comment (haraldF) [reply lesson19.md]: What's wrong with Muffins? :)

Comment (carlotrimarchi) [reply lesson19.md]: I was more worried by the equal sign inside the object 🤣

Comment (haraldF) [reply lesson19.md]: Since I'm very sensitive about muffins, you should always explicitly add that ;) anyway - fixed :)

**Pull request merged**